pkg/mover/chart.go

```diff
@@ -31,6 +31,19 @@ var (
 	ErrOCIRewritesMissing = errors.New("at least one rewrite rule is required")
 )
 
+type ChartLoadingError struct {
```

Comment:
I'd have preferred if we could have followed the naming structure layed out by the other two custom errors in this package, which start with `Err...`, something like `ErrLoadingChart`